### DISCUSSION TITLE
When creating a new branch, the name can't be changed until the page is refreshed

### DISCUSSION THREAD

**Discussion by airliftbluegill** [Able-Bear-Software/tab-nodes-tree#53] [Bug Report]
For example:

* "Create new branch as a child"
* Branch page automatically opens up
* Hit the edit button on the page next to the name, change "Branch" to "test", and hit enter
* Name still shows as "Branch" on the page and the Tab Nodes Tree sideview

To work around it:

* Refresh the branch page
* Now the name can be changed successfully in the same way

This happened on 1.3.0 and 1.4.0

**Comment by airliftbluegill** [comment]
Also, the "Add to branch" -> "New branch" feature doesn't show any children on the automatically-opened page, until it's refreshed.

**Comment by zeroarst** [comment]
Hey @airliftbluegill,
Thanks for bringing this up! I’m aware of the issue—it’s related to a data sync timing problem. I’ve already implemented a fix on my end, and it will be included in version 1.4.1, which should be released in the next few days. I should have created a ticket for it earlier, and I appreciate your patience!
Ticket: https://github.com/Able-Bear-Software/tab-nodes-tree/issues/55

**Comment by zeroarst** [comment]
Fixed in [1.4.1](https://github.com/Able-Bear-Software/tab-nodes-tree/discussions/59).
